src/api/spec/features/webui/requests/submissions_spec.rb

```diff
@@ -103,5 +103,56 @@
         expect(page).to have_text('In state new')
       end
     end
+
+    describe 'when under the beta program' do
+      before { Flipper.enable(:request_show_redesign, receiver) }
+
+      describe 'submit several packages at once against a factory staging project' do
+        let(:factory) { create(:project, name: 'openSUSE:Factory') }
+        let(:staging_workflow) { create(:staging_workflow, project: factory) }
+        let(:bs_request) do
+          create(:bs_request_with_submit_action,
+                 creator: receiver,
+                 target_project: factory,
+                 target_package: target_package,
+                 source_project: source_project,
+                 source_package: source_package)
+            .tap do |bs_request|
+            bs_request.staging_project = staging_workflow.staging_projects.first
+          end
+        end
+        let(:source_package) do
+          create(:package_with_files,
+                 name: 'package_a',
+                 project: source_project,
+                 changes_file_content: '- Fixes boo#1111101 CVE-2011-1101')
+        end
+        let(:target_package_b) { create(:package, name: 'package_b', project: factory) }
+        let(:another_bs_request_action) do
+          create(:bs_request_action_submit,
+                 bs_request: bs_request,
+                 source_project: source_project,
+                 source_package: source_package,
+                 target_project: factory,
+                 target_package: target_package_b)
+        end
+
+        before do
+          factory
+          staging_workflow
+          # Create another action to submit new files from different packages to package_b
+          another_bs_request_action
+        end
```

Comment:
I'd remove this before block and would use `let!` above to force to create these three values.